Core/Core/UIViews/CoreWebView.swift

```diff
@@ -257,6 +257,9 @@ open class CoreWebView: WKWebView {
                 document.addEventListener('load', checkSize, true)
             })
             window.addEventListener('error', checkSize, true)
+            if (window.ResizeObserver) {
+                new ResizeObserver(checkSize).observe(document.documentElement)
+            }
```

Comment:
This fixes a bug in the size I saw in a particular discussion. It's only iOS 13.4+, so I didn't check to see if it obsoletes any of the other mechanisms.